ballerina/metadata_types.bal

```diff
@@ -52,11 +66,43 @@ public type EntityFieldMetadata record {|
     RelationMetadata relation;
 |};
 
+# Represents the metadata associated with a field of an entity.
+# Only used by the generated persist clients and `persist:SQLClient`.
```

Comment:
```suggestion
# Only used by the generated persist clients and `persist:GoogleSheetsClient`.
```

Reply:
updated in [0639ae4](https://github.com/ballerina-platform/module-ballerina-persist/pull/211/commits/0639ae4b7a33d2a6f8534c4823231425175dea94)